src/build_settings.cpp

```diff
@@ -557,6 +559,14 @@ gb_global TargetMetrics target_openbsd_amd64 = {
 	str_lit("e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"),
 };
 
+gb_global TargetMetrics target_netbsd_amd64 = {
+	TargetOs_netbsd,
+	TargetArch_amd64,
+	8, 8, 8, 16,
```

Comment:
With LLVM 18 being merged in, this should probably be `8, 8, AMD_64_MAX_ALIGNMENT, 16`